src/ripple/rpc/handlers/AccountInfo.cpp

```diff
@@ -77,6 +77,19 @@ doAccountInfo(RPC::JsonContext& context)
     if (jvAccepted)
         return jvAccepted;
 
+    static constexpr std::
+        array<std::pair<std::string_view, LedgerSpecificFlags>, 9>
+            lsFlags{
+                {{"defaultRipple", lsfDefaultRipple},
+                 {"depositAuth", lsfDepositAuth},
+                 {"disableMasterKey", lsfDisableMaster},
+                 {"disallowIncomingXRP", lsfDisallowXRP},
+                 {"globalFreeze", lsfGlobalFreeze},
+                 {"noFreeze", lsfNoFreeze},
+                 {"passwordSpent", lsfPasswordSpent},
+                 {"requireAuthorization", lsfRequireAuth},
```

Comment:
This needs to support the new `DisallowIncoming` flags.

Reply:
Sure. Fixed.

Reply:
I would expect only to return those if the amendment is enabled.  It gives the false impression it those values are possible to set.

Reply:
Done.